client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,13 +1,19 @@
 import Grid from '@material-ui/core/Grid';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import CircularProgress from '@material-ui/core/CircularProgress';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
 import useStyles from './useStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
-import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
 import { useEffect } from 'react';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
 
-export default function Dashboard(): JSX.Element {
+interface Props {
+  children: React.ReactNode;
+}
+
+export default function Dashboard({ children }: Props): JSX.Element {
```

Comment:
Since we are using this component as a wrapper, maybe we can rename it to something like 'Layout'?

Reply:
Applied. Renamed ```Dashboard``` to ```Layout```